tests/standard_journey_logged_in.test.js

```diff
@@ -57,16 +58,15 @@ test('Standard journey is successful', async () => {
         await page.waitForSelector('#NextBtn')
         await page.click('#NextBtn')
 
-        //your-saved-address
```

Comment:
Do you have an address saved in your account? I have and this fails for me now.

Reply:
Ok I'll take another look and try to handle either scenario